lib/xebow/animation.ex

```diff
@@ -13,67 +13,30 @@ defmodule Xebow.Animation do
   """
 
   alias __MODULE__
-  alias Xebow.{AnimationFrame, RGBMatrix}
+  alias Xebow.{AnimationFrame}
```

Comment:
I don't think the curly braces are needed here anymore if there's only one module to import, unless I'm overlooking something.

Reply:
hmm I don't know how you saw this -- I definitely fixed this in one of my commits from yesterday. This is just `alias Xebow.Frame` on the latest commit.

Reply:
:thinking: Sorry, ignore me. I believe I was browsing through the commits, late at night, saw `alias Xebow.{AnimationFrame}` in one of them, and commented in the commit rather than at the end of the PR. My bad.